charts/s3gw/questions.yaml

```diff
@@ -2,207 +2,214 @@
 questions:
   # General settings
 
-  - variable: useCertManager
-    label: Use cert-manager
-    default: "true"
-    description: "Use cert-manager to provision TLS certificates"
+  - variable: serviceName
+    default: s3gw
+    description: "S3 Service Name"
+    label: "S3 Service Name"
+    required: true
+    type: string
+    group: "General"
+
+  - variable: ingress.enabled
+    default: true
+    description: "Deploy an Ingress (Required for TLS and UI)"
+    label: "Enable Ingress"
+    required: true
     type: boolean
     group: "General"
 
-  - variable: tls.publicDomain.crt
-    show_if: "useCertManager=false"
-    description: "S3 TLS certificate (Public Domain)"
-    label: "S3 TLS certificate (Public Domain)"
+  - variable: publicDomain
+    show_if: ingress.enabled=true
+    default: example.com
+    description: "Public domain of the S3 Service used by the Ingress"
+    label: "Public Domain"
+    required: true
     type: string
     group: "General"
 
-  - variable: tls.publicDomain.key
-    show_if: "useCertManager=false"
-    description: "S3 TLS key (Public Domain)"
-    label: "S3 TLS key (Public Domain)"
+  - variable: privateDomain
+    default: svc.cluster.local
```

Comment:
IMO, It is correct here to default to `svc.cluster.local` because this is the standard format inside k8s.